tools/markdown-magic/test/library-package-readme.md

```diff
@@ -1,8 +1,11 @@
-<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README:packageJsonPath=./package.json&devDependency=TRUE&scripts=TRUE) -->
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README:packageJsonPath=./package.json&devDependency=TRUE&scripts=TRUE&experimentalPackage=TRUE) -->
```

Comment:
Far-fetched idea: a policy handler could check that if a package is in the experimental scope that it includes the section in the README. Just trying to think of ways to keep everything up-to-date.

Reply:
That would be nice, yeah. There is a lot we could (and may want to) do to enforce certain invariants for our package READMEs, especially for our external-facing libraries.

Reply:
That also just made me think of a smaller improvement I could make. For the higher level library package pragma, we could (by default) inherit whether or not to include the notice based on the package scope. Let me give that a shot while I'm here.

Reply:
And done :) Change reflected in `pact-map`!